### PR TITLE
翻譯一小部分

### PR DESCRIPTION
**Pull request by RyanisyydsTT**

### CODE REVIEW

File: resources/views/admin/index.blade.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @endsection
 
 @section('content-header')
-    <h1>Administrative Overview<small>A quick glance at your system.</small></h1>
+    <h1>管理總覽<small>快速瀏覽您的主機.</small></h1>
```
Comment (MagicTeaMC) [inline resources/views/admin/index.blade.php]: system=系統

File: resources/views/admin/settings/advanced.blade.php [hunk 2]
```diff
@@ -25,34 +25,34 @@
...
+                                <label class="control-label">狀態</label>
                                 <div>
                                     <select class="form-control" name="recaptcha:enabled">
-                                        <option value="true">Enabled</option>
-                                        <option value="false" @if(old('recaptcha:enabled', config('recaptcha.enabled')) == '0') selected @endif>Disabled</option>
+                                        <option value="true">啟用</option>
+                                        <option value="false" @if(old('recaptcha:enabled', config('recaptcha.enabled')) == '0') selected @endif>停用</option>
                                     </select>
-                                    <p class="text-muted small">If enabled, login forms and password reset forms will do a silent captcha check and display a visible captcha if needed.</p>
+                                    <p class="text-muted small">如果啟用，登入表單和密碼重設表單將進行無聲驗證並在需要時顯示可見驗證。</p>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: 翻譯錯誤

File: resources/views/admin/settings/advanced.blade.php [hunk 3]
```diff
@@ -25,34 +25,34 @@
...
+                                    <p class="text-muted small">用於您的網站和 Google 之間的通信。請確保保密。</p>
                                 </div>
                             </div>
                         </div>
                         @if($showRecaptchaWarning)
                             <div class="row">
                                 <div class="col-xs-12">
                                     <div class="alert alert-warning no-margin">
-                                        You are currently using reCAPTCHA keys that were shipped with this Panel. For improved security it is recommended to <a href="https://www.google.com/recaptcha/admin">generate new invisible reCAPTCHA keys</a> that tied specifically to your website.
+                                        您當前使用的是與此面板一起發送的 reCAPTCHA 金鑰。為了提高安全性，建議 <a href="https://www.google.com/recaptcha/admin">生成新的隱形 reCAPTCHA 金鑰</a>，專門綁定到您的網站。
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: 翻譯錯誤

**Review by MagicTeaMC — CHANGES_REQUESTED** [review]

**Comment by RyanisyydsTT** [comment]
Fixed!

**Comment by RyanisyydsTT** [comment]
~~試試看Discord Ping可不可以在GitHub 運行w~~<@1170720196688871444>

File: resources/views/admin/settings/advanced.blade.php [hunk 4]
```diff
@@ -2,14 +2,14 @@
 @include('partials/admin.settings.nav', ['activeTab' => 'advanced'])
 
 @section('title')
-    Advanced Settings
+    進階設定
 @endsection
 
 @section('content-header')
-    <h1>Advanced Settings<small>Configure advanced settings for Pterodactyl.</small></h1>
+    <h1>進階設定<small>為 Pterodactyl 配置進階設定。</small></h1>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: 「設定」
「配置」是中國用語

File: resources/views/admin/settings/advanced.blade.php [hunk 5]
```diff
@@ -61,55 +61,55 @@
                 </div>
                 <div class="box">
                     <div class="box-header with-border">
-                        <h3 class="box-title">HTTP Connections</h3>
+                        <h3 class="box-title">HTTP 連接</h3>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: connection 在這邊翻譯成「連線」比較合適

File: resources/views/admin/settings/advanced.blade.php [hunk 6]
```diff
@@ -61,55 +61,55 @@
...
                     </div>
                     <div class="box-body">
                         <div class="row">
                             <div class="form-group col-md-6">
-                                <label class="control-label">Connection Timeout</label>
+                                <label class="control-label">連接逾時</label>
                                 <div>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php:70]: same

File: resources/views/admin/settings/advanced.blade.php [hunk 7]
```diff
@@ -61,55 +61,55 @@
...
+                                    <p class="text-muted small">在引發錯誤之前等待完成請求的時間量（以秒為單位）。</p>
                                 </div>
                             </div>
                         </div>
                     </div>
                 </div>
                 <div class="box">
                     <div class="box-header with-border">
-                        <h3 class="box-title">Automatic Allocation Creation</h3>
+                        <h3 class="box-title">自動分配創建</h3>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: pterodactyl 中的 allocation 其實是指主機的埠(port)

File: resources/views/admin/settings/advanced.blade.php [hunk 8]
```diff
@@ -61,55 +61,55 @@
...
+                                        <option value="false">停用</option>
+                                        <option value="true" @if(old('pterodactyl:client_features:allocations:enabled', config('pterodactyl.client_features.allocations.enabled'))) selected @endif>啟用</option>
                                     </select>
-                                    <p class="text-muted small">If enabled users will have the option to automatically create new allocations for their server via the frontend.</p>
+                                    <p class="text-muted small">如果啟用，用戶將有選擇通過前端自動為其服務器創建新分配的選項。</p>
                                 </div>
                             </div>
                             <div class="form-group col-md-4">
-                                <label class="control-label">Starting Port</label>
+                                <label class="control-label">起始端口</label>
```
Comment (MagicTeaMC) [inline resources/views/admin/settings/advanced.blade.php]: 「埠」
「端口」是中國用語

**Review by MagicTeaMC — CHANGES_REQUESTED** [review]

**Comment by RyanisyydsTT** [comment]
Done

**Comment by MagicTeaMC** [comment]
Any new updates?

**Comment by Ryanisyyds** [comment]
Maybe at Saturday or Sunday
好耶作業好多

**Comment by MagicTeaMC** [comment]
Well...okay

**Comment by RyanisyydsTT** [comment]
# Done List
- resourses/views/admin/api
- resourses/views/admin/databases
- resourses/views/admin/location
- resourses/views/admin/nests
- resourses/views/admin/settings
- resourses/views/admin/users
- resourses/views/admin/index.blade.php

# Unfinished List
- resourses/views/admin/eggs
- resourses/views/admin/mounts
- resourses/views/admin/nodes

**Comment by MagicTeaMC** [comment]
tested? or translated

**Comment by RyanisyydsTT** [comment]
Translated

**Comment by MagicTeaMC** [comment]
maybe time to merge it?

**Comment by RyanisyydsTT** [comment]
Merge first, got no free time .w.

**Pull request merged**